db/migrate/20200710101438_add_non_uk_gcse_columnns_to_application_qualification.rb

```diff
@@ -0,0 +1,8 @@
+class AddNonUkGcseColumnnsToApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_qualifications, :non_uk_qualification_type, :string
+    add_column :application_qualifications, :accrediting_country, :string
+    add_column :application_qualifications, :naric_reference_number, :string
+    add_column :application_qualifications, :comparable_uk_qualification, :string
```

Comment:
There's a separate PR for this, right?

Reply:
yea sorry i forgot to point it at the branch